specifyweb/frontend/js_src/lib/components/AppResources/Editor.tsx

```diff
@@ -236,6 +237,7 @@ export function AppResourceEditor({
                     ...resourceDirectory,
                     scope: getScope(resourceDirectory),
                   });
+                  globalThis.location.reload();
```

Comment:
page reload alone won't always apply the changes.
this is because the browser can cache the old app resource.
Thus, the cache must be cleaned before page is reloaded.

fortunately, you don't have to do this as it's already done on the xml-editor branch